internal/compiler/generator/cpp.rs

```diff
@@ -966,13 +966,15 @@ fn generate_item_tree(
         }),
     ));
 
+    // Statements will be overridden for repeated components!
     target_struct.members.push((
         Access::Private,
         Declaration::Function(Function {
             name: "subtree_index".into(),
-            signature: "(slint::private_api::ComponentRef /* component */) -> uintptr_t".into(),
+            signature: "([[maybe_unused]] slint::private_api::ComponentRef component) -> uintptr_t"
+                .into(),
             is_static: true,
-            statements: Some(vec!["/* unimplemented! */".to_owned(), "return 0;".into()]),
+            statements: Some(vec![format!("return {};", std::usize::MAX)]),
```

Comment:
How about?

```suggestion
            statements: Some(vec!["return numeric_limits<uintptr_t>::max();".into()]),
```

Maybe better than a huge 64bit constant

Reply:
Nah! That would be way too readable! I'll change it.